Polyhedron/demo/Polyhedron/Plugins/Surface_mesh/Mesh_simplification_plugin.cpp

```diff
@@ -153,8 +153,8 @@ void Polyhedron_demo_mesh_simplification_plugin::on_actionSimplify_triggered()
 
     if (selection_item)
       {
-        CGAL::Surface_mesh_simplification::Constrained_placement
-          <CGAL::Surface_mesh_simplification::Bounded_normal_change_placement
+        CGAL::Surface_mesh_simplification::internal::Constrained_placement
+          <CGAL::Surface_mesh_simplification::internal::Bounded_normal_change_placement
```

Comment:
The bounded normal change placement must check with the finite position. It pre-dates your changes, but still should be fixed. Please check other places too.